src/runtime_src/doc/toc/xrt_native_apis.rst

```diff
@@ -238,17 +245,17 @@ Code example of transferring data from the host to the device
 .. code:: c++
       :number-lines: 20    
            
-           auto input_buffer = xrt::bo(device, buffer_size_in_bytes, XCL_BO_FLAGS_NONE, bank_grp_idx_0);
+           auto input_buffer = xrt::bo(device, buffer_size_in_bytes, bank_grp_idx_0);
```

Comment:
Do not use `auto` when it makes the code longer...
I thought `bo` was renamed `buffer`.

Reply:
No, there will be no renaming.

Reply:
```c++
using buffer = xrt::bo;
```
?

Reply:
I considered that but decided against it.    I prefer some consistency and 'BO' for 'buffer object' is a pretty engrained concept in XRT.   There is also the C-APIs that use BO, which would have to be duplicated to Buffer.   It's too much of a mess and leaves inconsistent examples.  In hindsight, maybe buffer should have been used from the onset.  Next generation APIs ...

Reply:
OK. Anyway, for me it will be abstracted as a `sycl::buffer`, so the end-user will not see it...